docgen/src/gettingstarted.md

```diff
@@ -6,49 +6,323 @@ nav_groups:
 nav_sort: 0
 ---
 
-This is the home of `react-instantsearch`, a set of [React](https://facebook.github.io/react/) components to create unique instant-search experiences using [Algolia](https://www.algolia.com/) search engine.
+## Welcome to react-instantsearch
 
-**This is an alpha release, everything may be broken**.
+React-instantsearch is the ultimate toolbox for creating instant search
+experience using React.js and Algolia.
 
-Links:
-- [Chat with us](https://gitter.im/algolia/instantsearch.js) on gitter, we are active and waiting for feedback
```

Comment:
Can we keep those links visible somewhere obvious while we are still in alpha release? People need to understand that it's not released, and they need to come to the channel to give/get feedback/answers